app/forms/teacher_interface/work_history_form.rb

```diff
@@ -18,7 +19,8 @@ class WorkHistoryForm < BaseForm
     validates :school_name, presence: true
     validates :city, presence: true
     validates :country_code, presence: true
-    validates :email, presence: true, valid_for_notify: true
+    validates :contact_name, presence: true
```

Comment:
We'll need to add messages for these in the teacher_interface locale file.

Reply:
Good point, I've added some for these fields are the remaining fields in this form.